include/AudioListener.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef AUDIOLISTENER_H
+# define AUDIOLISTENER_H
+
+template <unsigned int N, class T>
+class Vect;
+
+class AudioListener
```

Comment:
The use of a class is questionable. Couldn't a namespace do the same job without giving the false feeling you can have multiple listeners?